tests/smoke/events.js

```diff
@@ -195,12 +195,12 @@
 			observer.events.push(evt);
 		}
 
-		editor.on('startSpellCheckOn', stdObserver);
-		editor.on('startScanWords', stdObserver);
-		editor.on('startCheckWordsAjax', stdObserver);
-		editor.on('startRender', stdObserver);
-		editor.on('spellCheckComplete', stdObserver);
-		editor.on('spellCheckAbort', stdObserver);
+		editor.on('startSpellCheckOn', stdObserver, null, null, -999);
```

Comment:
One is a value for `this`, the other is listenerData (kind of like standard params that are passed only into this listener for this event), and then the -999 is priority (very high) 

http://docs.ckeditor.com/#!/api/CKEDITOR.event-method-on